geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/SimpleFeaturePropertyAccessor.scala

```diff
@@ -24,7 +24,7 @@ trait SimpleFeaturePropertyAccessor extends LazyLogging {
     Thread.currentThread.setContextClassLoader(getClass.getClassLoader)
     try {
       import scala.collection.JavaConversions._
-      PropertyAccessors.findPropertyAccessors(sf, name, null, null).find(_.canHandle(sf, name, classOf[AnyRef]))
+      PropertyAccessors.findPropertyAccessors(sf, name, null, null).headOption
```

Comment:
note: `findPropertyAccessors` already calls `canHandle`, so calling it here was redundant